utils/prof_utils.py

```diff
@@ -0,0 +1,523 @@
+#!/usr/bin/env python3
+
+import numpy as np
+import re
+import sys
+import sys
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Reimport 'sys' (imported line 5)](https://app.codacy.com/app/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4645369)